tools/eslint/src/check-license.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ * Copyright 2024 Google Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const licenseHeader = `/**
+ * @license
+ * Copyright 2024 Google Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */\n\n`;
+
+/**
+ * @type import("eslint").Rule.RuleModule
+ */
+
+const enforceLicenseRule = {
+  // module.exports = {
+  meta: {
+    type: 'layout',
+    docs: {
+      description: 'Validate existence of license header',
+    },
+    fixable: 'code',
+    schema: [],
+    messages: {
+      licenseRule: 'Add license header.',
+    },
+  },
+  create(context) {
+    const sourceCode = context.sourceCode;
+    const comments = sourceCode.getAllComments();
+    const header = comments[0];
+
+    return {
+      Program(node) {
+        if (header?.value.includes('@license')) {
```

Comment:
We should check to include `@license`, `SPDX-License-Identifier: Apache-2.0` and
this RegEx `Copyright ([0-9]{4}) Google Inc.`. as we want a valid header as well.